Makefile

```diff
@@ -102,16 +102,13 @@ prepare-external:
 	git submodule update --init --recursive
 	nuget restore $(SOLUTION)
 	nuget restore Xamarin.Android-Tests.sln
-	$(foreach conf, $(CONFIGURATIONS), \
-		(cd external/xamarin-android-tools && make prepare CONFIGURATION=$(conf)) && \
-		(cd $(call GetPath,JavaInterop) && make prepare CONFIGURATION=$(conf) JI_MAX_JDK=8) && \
-		(cd $(call GetPath,JavaInterop) && make bin/Build$(conf)/JdkInfo.props CONFIGURATION=$(conf) JI_MAX_JDK=8) && ) \
-	true
+	(cd external/xamarin-android-tools && make prepare CONFIGURATION=$(CONFIGURATION)) && \
```

Comment:
There's no longer a need for `\`/line continuation.  These can all be separate invocations, and let `make` bail whenever any command fails.